templates/firefox-ios.xliff

```diff
@@ -858,10 +858,22 @@
         <source>Share Link</source>
         <note>Context menu item for sharing a link URL</note>
       </trans-unit>
+      <trans-unit id="ContextualHint.SearchBarPlacement.ExistingUsers" xml:space="preserve">
+        <source>Now you can move the toolbar to the bottom, so it's easier to enter info</source>
```

Comment:
Nit: this should be a "smart" (or "curly") apostrophe, so ’ rather than  ' : "it’s easier"
Note this should always be the case for the strings - but not needed for the comments

Reply:
I realize this is a pop-up: wanted to call out localizations are usually 30% longer than English (like German, French, Italian, Russian... actually most of of top tier locales)
Are we sure beforehand 30% longer strings will fit the pop up? thanks

Reply:
We've brought that point up to product and they seem okay with the string lengths. The popup also adjusts its size to fit text, so hopefully we won't get strange truncating behavior.

Reply:
Thanks for bringing it up :) That SGTM